### ISSUE TITLE
AWS Ignore tag "AutoTag_Creator"

### ISSUE THREAD

**Issue by ut0mt8** [hashicorp/terraform#31632]
TLDR; 
Every-time I create a tagged ressource in AWS with terraform I need to make a two time apply due to this specific tag being added by ? (Aws I guess) ; tag AutoTag_Creator. This is pretty annoying. I tried to exclude it with ignore tags but it doesn't seems to have an effect. 
btw  
```
lifecycle {
 ignore_changes = [tags]
  }
```
works but this is a far less ideal solution cause it can make drift happening on the other which is not what I want.

### Terraform Version
Latest
Terraform v1.2.7
on linux_amd64
+ provider registry.terraform.io/hashicorp/aws v4.25.0

### Terraform Configuration Files
```
provider "aws" {
  region = "eu-west-1"
  ignore_tags {
    keys = ["AutoTag_Creator"]
  }
}
```

### Debug Output
```
  # aws_subnet.subnets[1] will be updated in-place
  ~ resource "aws_subnet" "subnets" {
        id                                             = "subnet-07ce48c7a07d33fa1"
      ~ tags                                           = {
          - "AutoTag_Creator" = "arn:aws:iam::4916396832XX:user/xxx" -> null
            # (7 unchanged elements hidden)
        }
      ~ tags_all                                       = {
          - "AutoTag_Creator" = "arn:aws:iam::4916396832XX:user/xxx" -> null
            # (7 unchanged elements hidden)
        }
        # (14 unchanged attributes hidden)
    }
```

### Expected Behavior
I would terraform doing nothing ; ignoring this specific tag.

### Actual Behavior
Terraform wants to remove it.

**Comment by jbardin** [comment]
Hello,

This appears to be an issue or question with the AWS provider, not with Terraform itself. You can see existing issues and file a new one in their repository here: https://github.com/hashicorp/terraform-provider-aws/issues. If you have questions about Terraform or the AWS provider, it's better to use [the community forum](https://discuss.hashicorp.com/c/terraform-providers/tf-aws/33) where there are more people ready to help. The GitHub issues here are monitored only by a few core maintainers.

Thanks!
